src/global/mixins/mixins_spacing.scss

```diff
@@ -0,0 +1,104 @@
+/// Mixin to retrieve spacing values for the given breakpoints.
```

Comment:
Thanks for the input. Changed the way spacing is handeled but left the mixin implementation in `mixins_spacing.scss`, although unreferenced for the moment, until we have validated that the new solution will work for multiple cards since the cards are a bit different compared to the font-size (handling).

But the suggested solution is much simpler and therefore also more intuitive.